plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/utils/FactoryProjectImporter.java

```diff
@@ -132,6 +132,10 @@ public void apply(List<ProjectConfigDto> projectConfigs) throws OperationExcepti
                 Set<String> projectNames = new HashSet<>();
                 String createPolicy = factory.getPolicies() != null ? factory.getPolicies().getCreate() : null;
                 for (ProjectConfigDto projectConfig : projectConfigs) {
+                    if (projectConfig.getSource().getLocation() == null) {
+                        continue;
```

Comment:
mb we should show warning message that we can not import project because it doesn't contain source location?